testing/records/record_templates/main.tf

```diff
@@ -6,13 +6,6 @@ terraform {
   }
 }
 
-provider "leanspace" {
```

Comment:
Removing since we should only declare it once in the root module
(It was failing with the current config)